qtfred/ui/BriefingEditorDialog.ui

```diff
@@ -160,6 +160,13 @@
             </property>
            </widget>
           </item>
+		  <item row="0" column="0">
+		   <widget class="QLabel" name="iconScale">
+		    <property name="text">
+		     <string>Scale</string>
+		    </property>
+		   </widget>
+		  </item>
```

Comment:
This seems insufficient to me.
Doesn't there need to be some code that actually updates bi->scale for qtFRED, and not just the input field?

Reply:
Yes, this is correct. The briefing editor dialog for qtFRED currently has no code applied to it anywhere, just the window layout is done.

Reply:
Fair